### PR TITLE
Add ability to turn voice recognition on or off #173

### PR DESCRIPTION
**Pull request by pjaudiomv**

### CODE REVIEW

File: endpoints/functions.php [hunk 1]
```diff
@@ -1120,3 +1121,21 @@ function getIvrResponse($redirected_from = null, $prior_digit = null, $expected_
...
+    if (has_setting('speech_gathering') && json_decode(setting('speech_gathering'))) {
+        $inputType = "speech dtmf";
+    } else {
+        $inputType = "dtmf";
+    }
+    return $inputType;
+}
+
+function getPressWord() {
+    if (has_setting('speech_gathering') && json_decode(setting('speech_gathering'))) {
```
Comment (radius314) [inline endpoints/functions.php]: can you use a ternary operator?

File: endpoints/zip-input.php [hunk 2]
```diff
@@ -11,12 +11,18 @@
         $action = "address-lookup.php";
     }
 
+    if (has_setting('speech_gathering') && json_decode(setting('speech_gathering'))) {
```
Comment (radius314) [inline endpoints/zip-input.php]: ternary operator plz?

File: endpoints/functions.php [hunk 3]
```diff
@@ -1123,19 +1123,11 @@ function getIvrResponse($redirected_from = null, $prior_digit = null, $expected_
 }
 
 function getInputType() {
-    if (has_setting('speech_gathering') && json_decode(setting('speech_gathering'))) {
-        $inputType = "speech dtmf";
-    } else {
-        $inputType = "dtmf";
-    }
+    $inputType = has_setting('speech_gathering') && json_decode(setting('speech_gathering')) ? "speech dtmf" : "dtmf";
```
Comment (radius314) [inline endpoints/functions.php]: You can just return rather than assign to a new variable and return.

Comment (pjaudiomv) [reply endpoints/functions.php]: Cool wasnt sure about that, makes sense.

**Pull request merged**